components/Act/ActAdd.js

```diff
@@ -10,7 +10,7 @@ import { connect } from 'react-redux'
 
 // todo if image is not present then use a fallback.
 const ActAdd = ({ roles, ...props }) => {
-  if (roles && roles.includes('activity-provider')) {
+  if (roles && roles.includes('activityProvider')) {
```

Comment:
Please replaces the literal string with Role.ACTIVITY_PROVIDER. from person.constants.